flink-runtime/src/test/java/org/apache/flink/runtime/testutils/CommonTestUtils.java

```diff
@@ -310,17 +311,29 @@ public static void waitForSubtasksToFinish(
     }
 
     /** Wait for at least one successful checkpoint. */
```

Comment:
The comment is out of date once you introduce the parameter of `numCheckpoints`.

Reply:
I've update the comment.